tests/web3js/eth_non_interactive_test.js

```diff
@@ -1,112 +1,132 @@
 const web3Utils = require('web3-utils')
 const { assert } = require('chai')
 const conf = require('./config')
+const helpers = require('./helpers')
 const web3 = conf.web3
 
-it('get chain ID', async() => {
-    let chainID = await web3.eth.getChainId()
-    assert.isDefined(chainID)
-    assert.equal(chainID, 646n)
+it('get chain ID', async () => {
+  let chainID = await web3.eth.getChainId()
+  assert.isDefined(chainID)
+  assert.equal(chainID, 646n)
 })
 
 it('get block', async () => {
-    let height = await web3.eth.getBlockNumber()
-    assert.equal(height, conf.startBlockHeight)
-
-    let block = await web3.eth.getBlock(height)
-    assert.notDeepEqual(block, {})
-    assert.isString(block.hash)
-    assert.isString(block.parentHash)
-    assert.lengthOf(block.logsBloom, 514)
-    assert.isDefined(block.timestamp)
-    assert.isTrue(block.timestamp >= 1714413860n)
-
-    let blockHash = await web3.eth.getBlock(block.hash)
-    assert.deepEqual(block, blockHash)
-
```

Comment:
why all these spacing changes?

Reply:
It comes from saving the file on my editor, due to the JS formatting options I have :sweat_smile:

Reply:
Can you change to 4 so we have it consistent as we have in other places (go). We should add a linter rule for it, but for now if you can just change to 4 so we don't keep having this chnages.

Reply:
Sounds good :+1: 
I have reverted the formatting changes in this commit: https://github.com/onflow/flow-evm-gateway/pull/253/commits/81da50167be46dc469eb68c96f58466b4998c791